config/locales/codelists/2_03/iati.en.yml

```diff
@@ -43,7 +43,8 @@ en:
       '1': Yes - China
       '2': Yes - India
       '3': Yes - China and India
-      '4': 'No'
+      '4': No - This activity benefits China or India but GDI does not apply
```

Comment:
Isn't this change splitting the single "No" answer into two?

A number of existing activities that have GDI set to `4` should now presumably be `5`, but we have no way of knowing.

Reply:
It is kind of a split. It defines the "No" answer better. Option 4 will be for activities that benefit China/India but were created before the GDI policy existed (not sure about when that policy was introduced). Option 5 is for those activities that benefit any other countries that aren't China or India. 

It is a very good point and a good reason to be concerned. Assuming users stated "No" on activities benefitting other countries than China or India, I could swap the values for 4 and 5. But if they used the former option "No" for activities benefitting China or India _before_ the GDI policy, these activities would be allocated the wrong code :/

Reply:
Probably worth flagging with @mec just to be on the safe side.

Reply:
It's a good point, thanks for raising it. 

It looks like this is the field where we ingested 'replace me' - is that correct?

As with most cases like this BEIS have introduce the new response and will have to decide how to handle the transition of data - I'll try and pick this up with them this morning.

Let's hold this PR until we can tease a little more out of BEIS - ultimately they will have to decide if they can accept the blanket no as I don't think there is any other option right now.

Reply:
The decision made by the client is to map any existing "No" option to "GDI not applicable". For this, I have decided to swap the values of codes 4 and 5. This saves us a migration, since existing activities with value "No" (code 4), will now keep their code, which equals to "GDI not applicable". I've pushed the new code again and I'll be requesting a re-review